WireDomain/Sources/WireDomain/Notifications/Builders/Conversation/ConversationCallingEventNotificationBuilder.swift

```diff
@@ -37,16 +37,25 @@ struct ConversationCallingEventNotificationBuilder: ConversationCallingEventNoti
             return nil
         }
 
+        var resolvedConversationID: ConversationID {
+            let callingConversationID = calling.qualifiedConversationID
+            guard !callingConversationID.id.isEmpty,
+                  let conversationUUID = UUID(uuidString: callingConversationID.id)
+            else {
+                return conversationID
```

Comment:
wouldn't this cause the same issue as you described, where the conversation ID is the id of the self conversation ? If so, should we treat this as an error instead ?